src/Microsoft.AspNet.Http.Abstractions/Extensions/HttpResponseWritingExtensions.cs

```diff
@@ -16,10 +16,10 @@ public static class HttpResponseWritingExtensions
         /// <summary>
         /// Writes the given text to the response body. UTF-8 encoding will be used.
         /// </summary>
-        /// <param name="response"></param>
-        /// <param name="text"></param>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
+        /// <param name="response">The <see cref="HttpResponse"/>.</param>
+        /// <param name="text">The text to write to the response.</param>
+        /// <param name="cancellationToken">Notifies when request operations should be cancelled.</param>
```

Comment:
This is `HttpResponseWritingExtensions` and it says `request operations` in the doc comment.